src/components/Alert.tsx

```diff
@@ -109,13 +117,15 @@ export class Alert extends React.Component<IAlertProps> {
           size="small"
           onClick={() => {
             showDialog({
-              title: 'Detailed message',
-              body: this.props.details,
-              buttons: [Dialog.okButton({ label: 'DISMISS' })]
+              title: this.props.trans.__('Detailed message'),
+              body: this.props.trans.__(this.props.details),
```

Comment:
So this needs to be translated on the source, since we don't know the content of `this.props.details` wherever that string os coming from it should be handled there and just displayed here.

Reply:
Good point. Looks like the body of this can be translated from the git extension since the error message seems to come from elsewhere